tests/debugger/test_debugger_telemetry.py

```diff
@@ -0,0 +1,107 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+import tests.debugger.utils as debugger
+from utils import scenarios, features, missing_feature, context
+from utils.tools import logger
+from tests.test_telemetry import load_telemetry_json, get_lang_configs
```

Comment:
can you move `oad_telemetry_json` and `get_lang_configs` inside `tests.debugger.utils`? otherwise, the CI may miss scenario to run when those function are modified.

Reply:
Those methods are also used by `tests/tests_telemetry.py`, and rely on the `tests/telemetry_intake/static` files. Do you mean I should import them into `tests.debugger.utils` or I should duplicate them?

Reply:
Yup sure thing